### PR TITLE
Make sigmoid_binary_cross_entropy work for infinite logits.

### PR DESCRIPTION
**Pull request by jondeaton**

### CODE REVIEW

**Comment by jondeaton** [comment]
The proposed implementation does not solve the issue.
